jerry-core/ecma/builtin-objects/ecma-builtin-date.cpp

```diff
@@ -324,25 +325,118 @@ ecma_builtin_date_now (ecma_value_t this_arg) /**< this argument */
 /**
  * Handle calling [[Call]] of built-in Date object
  *
+ * See also:
+ *          ECMA-262 v5, 15.9.2.1
+ *
  * @return completion-value
  */
 ecma_completion_value_t
-ecma_builtin_date_dispatch_call (const ecma_value_t *arguments_list_p, /**< arguments list */
-                                 ecma_length_t arguments_list_len) /**< number of arguments */
+ecma_builtin_date_dispatch_call (const ecma_value_t *args_list_p, /**< arguments list */
+                                 ecma_length_t args_len) /**< number of arguments */
 {
-  ECMA_BUILTIN_CP_UNIMPLEMENTED (arguments_list_p, arguments_list_len);
+  /* FIXME:
+   *       Fix this, after Date.prototype.toString is finished.
+   */
+  return ecma_builtin_date_dispatch_construct (args_list_p, args_len);
 } /* ecma_builtin_date_dispatch_call */
 
 /**
  * Handle calling [[Construct]] of built-in Date object
  *
+ * See also:
+ *          ECMA-262 v5, 15.9.3.1
+ *
  * @return completion-value
  */
 ecma_completion_value_t
-ecma_builtin_date_dispatch_construct (const ecma_value_t *arguments_list_p, /**< arguments list */
-                                      ecma_length_t arguments_list_len) /**< number of arguments */
+ecma_builtin_date_dispatch_construct (const ecma_value_t *args_list_p, /**< arguments list */
+                                      ecma_length_t args_len) /**< number of arguments */
```

Comment:
In UTC it is called `args` instead of `arguments`, I wanted to use more consistent variable naming. I can revert it if it bothers you.